### ISSUE TITLE
Steadler pencil eraser not working

### ISSUE THREAD

**Issue by digitaldazz** [mb1986/rm-hacks#46]
Steadler pencil eraser not working with rm-hacks. 
I have not tried other versions but it used to work on rm1: 2.15.1.1189.

Thank you for the hack, still very useful.

**Comment by mb1986** [comment]
I think I've found it: https://github.com/ddvk/remarkable-stylus/blob/master/qevdevtablethandler.cpp#L82

Shouldn't be hard to add it.

**Comment by mb1986** [comment]
@digitaldazz, could you please check if after installing v0.0.6 the eraser does work?

```shell
sh -c "$(wget https://raw.githubusercontent.com/mb1986/rm-hacks/main/install.sh -O-)" _ patch 0.0.6
```

**Comment by digitaldazz** [comment]
Hi, just tried and it does not switch to the eraser.

**Comment by digitaldazz** [comment]
Interesting, when on the eraser and I use the steadler eraser it switches to the pen.

**Comment by mb1986** [comment]
I've done some changes, could you please re-install patch and check again?

```shell
sh -c "$(wget https://raw.githubusercontent.com/mb1986/rm-hacks/main/install.sh -O-)" _ patch 0.0.6
```

Don't forget to answer `y` to install `ddvk/remarkable-stylus` and restart whole device.

**Comment by digitaldazz** [comment]
Did a full power off and restart, the eraser is still drawing on pen mode and when I put it in eraser mode it switches to the pen mode.
pressed y to install ddvk.

**Comment by mb1986** [comment]
Well, I'd need to buy the pen to test it ;)

**Comment by digitaldazz** [comment]
Ahh, ok. It is definetly triggering to return to pen mode after erasing, so that part is working. It is just the initial trigger to activate eraser mode that is not working.

Thank you for trying.

**Comment by mb1986** [comment]
@digitaldazz, I've changed yet another thing. Could you please verify?

```shelll
sh -c "$(wget https://raw.githubusercontent.com/mb1986/rm-hacks/main/install.sh -O-)" _ patch 0.0.6
```

**Comment by digitaldazz** [comment]
Hi, I just tried it and it did not trigger the eraser mode and the eraser mode still switched back to pen when the steadler eraser is used to erase.

**Comment by mb1986** [comment]
Thanks, I'll try one more thing tomorrow.

**Comment by digitaldazz** [comment]
Thank you, Michal.

**Comment by mb1986** [comment]
Could you please give it another try?

```shell
sh -c "$(wget https://raw.githubusercontent.com/mb1986/rm-hacks/main/install.sh -O-)" _ patch 0.0.6
```

**Comment by digitaldazz** [comment]
It reacts the same as the previous ones.

**Comment by mb1986** [comment]
Thanks, I'll work on that again when I buy the pen.

**Comment by digitaldazz** [comment]
OK thank you, Michal.

**Comment by mb1986** [comment]
@digitaldazz, please find the confirmation that the Staedtler Jumbo works just fine https://github.com/mb1986/rm-hacks/pull/49#issuecomment-1826827726. Can't understand why one pen works while the other don't (?).

**Comment by ingatellent** [comment]
I can confirm that the eraser on the Kindle Scribe Premium pen also _does not work_ on my device with version 0.0.6. This is on a **rm1**, on a **rm2** all erasers should work out of the box on stock software (this might be what others has tested?).

**Comment by Eeems** [comment]
> I can confirm that the eraser on the Kindle Scribe Premium pen also _does not work_ on my device with version 0.0.6. This is on a **rm1**, on a **rm2** all erasers should work out of the box on stock software (this might be what others have tested?).

To confirm, this is after saying yes to install the plugin for the lamy pen as part of installing rm-hacks?

**Comment by ingatellent** [comment]
> To confirm, this is after saying yes to install the plugin for the lamy pen as part of installing rm-hacks?
Yes, I answered yes to installing the Lamy hack as well. The Kindle scribe pen has both a side button and a rubber eraser at the top. The side button works perfectly with the lamy hack. The rubber eraser just works as a stylus/pen.

**Comment by digitaldazz** [comment]
> @digitaldazz, please find the confirmation that the Staedtler Jumbo works just fine [#49 (comment)](https://github.com/mb1986/rm-hacks/pull/49#issuecomment-1826827726). Can't understand why one pen works while the other don't (?).

Maybe I should reset the device and try on a fresh install.

**Comment by mb1986** [comment]
Like @ingatellent mentioned earlier it may be related to rM1 vs rM2. Frankly speaking, I don't know.

**Comment by WehZet** [comment]
Ok, didn't know. For me it works as stock and I find it OK this way

**Comment by dusterdieck** [comment]
Confirming what others have said, Noris Jumbo works appropriately on PDFs but not in notebooks.

rm1 v3.9.3.1986

rM Hacks v0.0.8 (also tested on 0.0.9 alpha.10)

**Comment by haydensp** [comment]
@digitaldazz did you ever solve this? I just updated everything to 0.0.9, and I'm running into the same issue. RM1, eraser was working great with ddvk hacks on 2.15.1.1189 but it's not triggering the eraser in the rm-hacks current version...

**Comment by atngames** [comment]
Hi, I have the Norris jumbo and it always worked in all the documents.
Is it a rm1 problem only?
Could you tell which rm version for those who have bugs?

**Comment by ingatellent** [comment]
Yes it is an RM1 issue only - it's because the reMarkable team chose not to recognize the rubber button tool one the RM1, probably since they never advertised a pen with eraser for the RM1.

Status right now is that it rmhacks enables the rubber eraser but it only works on pdf-pages and not on notepages.

As mentioned in #279 it sometimes works on notepages when using splitdoc.

I suspected, that it was due to the possibilty of adding text to a notepage, but hiding the text icon which is possible in the latest alpha chances nothing, it still doesn't work on notepages on RM1.

**Comment by atngames** [comment]
Okay thank you for all this input !

**Comment by ingatellent** [comment]
For what it's worth:
I found the reason for the rubber eraser is not working in notebooks on the rM1:
The shortcut Ctlr+I which the plugin generates is used by xochitl for toggling italic text. Changing the shortcut to something else (Ctrl+T?) would make it usable also in notebooks.
